decisions/2019/11/tapps-meeting-remote-errors-etc.md

```diff
@@ -0,0 +1,128 @@
+# 2019-11-11 @dhis2/team-apps Meeting
+
+## Attending
+
+Viktor, Austin, Jan-Gerke, Ismay, Hendrik, Birk, Erik
+
+## Maintenance App Refactor
+
+We are ready to work on it and have started prototyping: https://github.com/dhis2/mar-y/
+
+Not currently a priority, but it is something we need to spend time on when we
+can.
+
+Getting it up to a technical standard we can maintain is the first priority,
+switching out UI components to the DHIS2 ones.
+
+When it comes to improvements and functional changes to the Maintenance
+application, it is a bit stickier, as any changes to the Maintenance
+application triggers a lot of work related to updating training materials and
+training for people. We should gather requirements and write specs for changes
+we think would be inline with our way of work.
+
+Input from the field from Austin:
+
+- Import/Export app cannot import GML
```

Comment:
You mean Maintenance App here, right?

Reply:
I think it's actually: Import/Export app can't import polyglyphs (is that the right term?) (which is a concern @amcgee raised, if I'm not mistaken)

Reply:
The problem that was raised in the meeting was that the Mainteanance app does not support selecting or uploading polygons for organisation units. Only manually entered coordinates are supported. That's why the import/export app is used to import GML/polygons.
I agree that this is something that should be supported in the maintenance app, but it's pretty complex if you want maps support, so this is a feature that would benefit greatly from cross-team cooperation with the GIS team. But we will probably not introduce this until the refactor is feature-complete.

Reply:
Ah I see, my bad. You're right and I agree with your suggestion

Reply:
Good catch, @Birkbjo! My notes were inconclusive and I had this under the wrong section. I'll update.

Reply:
933100c